nucleotide-count/example.py

```diff
@@ -7,6 +7,7 @@ def count(strand, abbreviation):
 
 
 def nucleotide_counts(strand):
+    [_validate(x) for x in strand]
```

Comment:
List comprehensions are meant to be used with functions having return values. Just use a `for` loop.